### PR TITLE
Nicole Tan 9-10 SPS

### PR DESCRIPTION
**Pull request by Hikaruimouto**
Please fill out the survey before submitting the pull request. Thanks!

🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀

How many hours did you spend on this assignment?
2 hours 

Please fill in one error and/or error message you received while working on this assignment.
If statement didn't work due to conflicting terms. 

What part of the assignment did you spend the most time on?
Debugging 

Comfort Level (1-5):
3

Completeness Level (1-5):
3

What did you think of this deliverable?
Okay 

Is there anything in this code that you feel pleased about?
All good.

### CODE REVIEW

**Comment by Hikaruimouto** [comment]
Please fill out the survey before submitting the pull request. Thanks!

🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀

How many hours did you spend on this assignment?
2 hours

Please fill in one error and/or error message you received while working on this assignment.
My score counted from 0 instead of 1. 

What part of the assignment did you spend the most time on?
Debugging 

Comfort Level (1-5):
3

Completeness Level (1-5):
3

What did you think of this deliverable?
Okay

Is there anything in this code that you feel pleased about?
It works.

**Comment by Hikaruimouto** [comment]
🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀

How many hours did you spend on this assignment?
I lost count

Please fill in one error and/or error message you received while working on this assignment.
Game did not go into the function statements

What part of the assignment did you spend the most time on?
Debugging 

Comfort Level (1-5):
3

Completeness Level (1-5):
3

What did you think of this deliverable?
Okay

Is there anything in this code that you feel pleased about?
That it works lol

File: script.js [hunk 1]
```diff
@@ -1,4 +1,160 @@
...
+      totalPlays +
+      " times";
+  }
+  if (userChoose == "score check") {
+    myOutputValue =
+      " You have won " + userScore + " out of " + totalPlays + " times";
+  }
+  return myOutputValue;
+};
+
```
Comment (JustinWong98) [inline script.js:76]: Good job combining if statements with similar outcomes into one statement!

File: script.js [hunk 2]
```diff
@@ -1,4 +1,160 @@
...
+  }
+  console.log("Playing " + currentGameMode);
+  // Run the specific game the user wants
+
+  if (currentGameMode == "normal") {
+    myOutputValue = normalSPS(input);
+  }
+  if (currentGameMode == "reverse") {
+    myOutputValue = reverseSPS(input);
+  }
```
Comment (JustinWong98) [inline script.js:153]: perhaps you could combine the latter lines of code with the former ones? For example:

if (input == "normal") {
    currentGameMode = "normal";
myOutputValue = normalSPS(input);
  } else {
    currentGameMode = "reverse";
myOutputValue = reverseSPS(input);
  }

File: script.js [hunk 3]
```diff
@@ -1,4 +1,160 @@
...
+// ///  Checking your score ///
+// var scoreCheck = function () {
+//   myOutputValue =
+//     " You have won " + userScore + " out of " + totalPlays + " times";
+// };
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  var myOutputValue = "";
+  input = input.toLowerCase();
```
Comment (JustinWong98) [inline script.js:129]: Good job with the .toLowerCase() to standardize input!

**Review by JustinWong98 — COMMENTED** [review]
Very clear and easy to understand code overall! Good job :)

**Pull request closed without merging**